src/ol_openedx_git_auto_export/README.md

```diff
@@ -0,0 +1,57 @@
+#### Steps to install on edX platform:
+
+
+You can install this plugin into any Open edX instance by using any of the following methods:
+
+
+**Option 1: Install from PyPI**
+
+.. code-block::
+
+    # If running devstack in docker, first open a shell in CMS (make studio-shell)
+
+    pip install ol-openedx-git-auto-export
+
+
+**Option 2: Build the package locally and install it**
+
+Follow these steps in a terminal on your machine:
+
+1. Navigate to ``open-edx-plugins`` directory
+2. If you haven't done so already, run ``./pants build``
+3. Run ``./pants package ::``. This will create a "dist" directory inside "open-edx-plugins" directory with ".whl" & ".tar.gz" format packages for all the "ol_openedx_*" plugins in "open-edx-plugins/src")
+4. Move/copy any of the ".whl" or ".tar.gz" files for this plugin that were generated in the above step to the machine/container running Open edX (NOTE: If running devstack via Docker, you can use ``docker cp`` to copy these files into your LMS container)
+5. Run a shell in the machine/container running Open edX, and install this plugin using pip
+
+Configuration
+------------
+**1) edx-platform configuration**
+
+- Enable the following FEATURES flags in the config file in Open edX. For any release after Juniper, that config file is ``/edx/etc/cms.yml``. If using `private.py`, you will need to enable these FEATURES in `cms/envs/private.py`.
+
+  ```
+  "FEATURES": {
+    "ENABLE_EXPORT_GIT": true,
+  }
+  ```
+- Set your commit user in `cms/envs/common.py`, if you don't want to use the default one
+```
+GIT_EXPORT_DEFAULT_IDENT = {
+    'name': 'STUDIO_EXPORT_TO_GIT',
+    'email': 'STUDIO_EXPORT_TO_GIT@example.com'
+}
+```
+- Now run to install `pip install ol-openedx-git-auto-export.whl`. So it means just install the whl file created using pants build.
```

Comment:
I mentioned this in the previous review too, The installation general guide has already been mentioned in https://github.com/mitodl/open-edx-plugins/pull/36/files#diff-20e4582e4d639837b6986781a3f4747a8702d79ecb00783d9f235ad41648c95aR11-R24. We don't need to duplicate it here.